phoenix-core/src/test/java/org/apache/phoenix/query/PhoenixTestBuilder.java

```diff
@@ -368,7 +368,7 @@ public void upsertRow(int rowIndex) throws SQLException {
      * for specific test requirements.
      */
     public static class SchemaBuilder {
-        private static final AtomicInteger TENANT_COUNTER = new AtomicInteger(0);
+        private static final AtomicInteger TENANT_COUNTER = new AtomicInteger(7);
```

Comment:
is there a specific reason that we want to have an initial integer `7` instead of `0`?

Reply:
Oh!!, It was a leftover from my testing, will reset it to 0